ui/src/app/views/account-creation/component.ts

```diff
@@ -39,12 +43,16 @@ export class AccountCreationComponent {
   accountCreated: boolean;
   conflictError = false;
   usernameCheckTimeout: NodeJS.Timer;
+  backgroundImgSrc = '/assets/images/create-account-male@2x.jpg';
 
   constructor(
     private profileService: ProfileService,
     private signInService: SignInService,
-    private invitationKeyService: InvitationKeyComponent
-  ) {}
+    private invitationKeyService: InvitationKeyComponent,
+    private appComponent: AppComponent
+  ) {
+    this.appComponent.backgroundImgSrc = this.backgroundImgSrc;
```

Comment:
is `this.backgroundImgSrc` used anywhere else? Or could we just use the hardcoded path here, and remove the variable in the component?